src/reynir_correct/settings.py

```diff
@@ -478,9 +488,160 @@ def add(morph: str, boundlist: List[str], freelist: List[str]) -> None:
         Morphemes.FREE_DICT[morph] = freelist
 
 
+class Ritmyndir:
+
+    # dict { wrong_word_form : (lemma, id, cat, correct_word_form, tag, eink, malsnid, stafs, aslatt, beyg) }
+    # þurrð;10963;kvk;þurðar;þurrðar;EFET;0;URE;;;;1745-1745;KLIM
+    # þurrka;425063;so;þurkaði;þurrkaði;;4;VILLA;R4RR;;;;SKOLAVERK
+    DICT: Dict[str, RitmyndirTuple] = dict()
+
+    @staticmethod
+    def contains(word: str) -> bool:
+        """Check whether the word form is in the Ritmyndir dictionary"""
+        d = Ritmyndir.DICT
+        return word in d or word.lower() in d
+
+    @staticmethod
+    def get_lemma(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_id(wrong_form: str) -> int:
```

Comment:
OK, I suggest that all of these functions call a single helper function taking the index as a parameter.

Reply:
Overhauled all get-functions.